client/js/utils/dom-events.js

```diff
@@ -1,10 +1,12 @@
 import fromEvent from 'xstream/extra/fromEvent';
 import debounce from 'xstream/extra/debounce';
 import throttle from 'xstream/extra/throttle';
+import xs from 'xstream';
 
 export const onWindowResizeDebounce$ = fromEvent(window, 'resize').compose(debounce(500));
 export const onWindowScrollThrottle$ = fromEvent(window, 'scroll').compose(throttle(100));
 export const onWindowScrollDebounce$ = fromEvent(window, 'scroll').compose(debounce(500));
+export const onWindowScrollThrottleDebounce$ = xs.merge(onWindowScrollThrottle$, onWindowScrollDebounce$);
```

Comment:
Yea, but seeming to have a debounced throttle event means we're going to get neither, it seems a hack to something that might have a better solution.

I'll have a poke about.